conans/client/generators/cmake_find_package_multi.py

```diff
@@ -256,6 +268,19 @@ class CMakeFindPackageMultiGenerator(CMakeFindPackageGenerator):
                          $<$<CONFIG:{{config}}>:{{ '${'+pkg_name+'_COMPONENTS_'+config.upper()+'}'}}>
                          {%- endfor %})
         endif()
+
+        ########## BUILD MODULES ####################################################################
+        #############################################################################################
+
+        ########## GLOBAL BUILD MODULES #############################################################
+
+        {%- for config in configs %}
+
+        foreach(_BUILD_MODULE_PATH {{ '${'+pkg_name+'_BUILD_MODULES_PATHS_'+config.upper()+'}' }})
```

Comment:
I wouldn't mind if build_modules does not work/exist at all for the multi-config. In fact, this is something I think we should deprecate in Conan 2.0, the complexity is high, the use cases are probably not that legit anymore (the multi-config packages were created when multi-generators didn't exist)